data/events.yml

```
@@ -1,4 +1,12 @@
-- lang: pt
+lang: en
+  startDate: '2025-02-22'
+  endDate: '2025-02-22'
+  location: Gurugram , India
+  speaker: 'To be announced'
+  title: 'UDAAN 2025'
+  subject: 'Empowering Innovation Through Technology'
+  url: https://www.commudle.com/communities/gurugram-kotlin-user-group/events/udaan-2025
+  - lang: pt
```

Thanks for the contribution!
The check failed because the formatting is broken.
Here's the fix
```suggestion
- lang: en
  startDate: '2025-02-22'
  endDate: '2025-02-22'
  location: Gurugram , India
  speaker: 'To be announced'
  title: 'UDAAN 2025'
  subject: 'Empowering Innovation Through Technology'
  url: https://www.commudle.com/communities/gurugram-kotlin-user-group/events/udaan-2025
- lang: pt
```